src/sentry/newsletter/base.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import
+
+
+class Newsletter(object):
+    PRODUCT_CAMPAIGN = 1
```

Comment:
Yeah, then maybe I'm using the wrong terminology. Would it make more sense if I called this "list"? Because, yeah, this literally means "what I am subscribing for."